deco-sites/otica-isabela
# hide breadcrumb on mobile

## What is this contribution about?

Hided Breadcrumb on mobile view.

## How to test it?

Go to a PLP and check the breadcrumb on each device, Mobile / Tablet and Desktop.

## Extra info
PS: It's visible only on Tablet and Desktop, Mobile devices is hidden.
